src/components/main/settings/mod.rs

```diff
@@ -18,7 +18,6 @@ pub struct Props {
 
 #[allow(non_snake_case)]
 pub fn Settings(cx: Scope<Props>) -> Element {
-    log::debug!("rendering Settings");
```

Comment:
I think this should stay. it won't get logged unless uplink is started with a specific environment variable. It's possible to disable these at compile time as well: https://docs.rs/log/latest/log/index.html#compile-time-filters

Reply:
Good point!